### PR TITLE
Ronin fight AI skirmishes farther away

### PR DESCRIPTION
**Pull request by sprunk**

### CODE REVIEW

File: LuaRules/Configs/tactical_ai_defs.lua [hunk 1]
```diff
@@ -1369,14 +1369,15 @@ local behaviourConfig = {
 		fightOnlyUnits = medRangeExplodables,
 		maxSwarmLeeway = 30,
 		minSwarmLeeway = 130,
-		jinkPeriod = 2,
+		jinkPeriod = 2.5,
```
Comment (sprunk) [inline LuaRules/Configs/tactical_ai_defs.lua:1372]: As far as I can tell, only integers make sense here and this is equivalent to 3.
```suggestion
		jinkPeriod = 3,
```

**Comment by palimpsestate** [comment]
Tidied that up.  Any further input?

**Comment by sprunk** [comment]
From a technical PoV looks good, now GoogleFrog should say something from a design/gameplay PoV.

**Comment by palimpsestate** [comment]
*crickets*

**Comment by GoogleFrog** [comment]
I told a Ronin to attack move a Jack and it walked far away while reloading.
image

**Comment by GoogleFrog** [comment]
Also Ronin very rarely fires at a Jack that is running from it.

**Pull request merged**